datalad/support/annexrepo.py

```diff
@@ -313,37 +313,46 @@ def get_file_key(self, files):
                 # Not sure, whether or not this can actually happen
                 raise e
 
-        return output[0].split(linesep)[0]
+        return output[0].rstrip(linesep).split(linesep)
 
 
     @normalize_paths
     def file_has_content(self, files):
-        """ Check whether `files` are present with their content.
+        """ Check whether files have their content present under annex.
 
         Parameters:
         -----------
-        files: list
+        files: list of str
             file(s) to check for being actually present.
 
         Returns:
         --------
-        list of (str, bool)
-            list with a tuple per file in `files`.
+        list of bool
```

Comment:
If necessary you can anyways zip result with original list to get the same rafting as before.
But as you could see from all of its usages so far, you never needed that yet

On April 21, 2015 3:52:00 AM EDT, bpoldrack <notifications@github.com> wrote:
>>              file(s) to check for being actually present.
>>  
>>          Returns:
>>          --------
>> -        list of (str, bool)
>> -            list with a tuple per file in `files`.
>> +        list of bool
>
>From my point of view, the former list of tuples is more convenient,
>since you can easily access the entry corresponding to certain files,
>whereas you now have to match indices to find it, but that's a matter
>of taste, I guess.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/datalad/datalad/pull/120/files#r28756561